app.vue

```diff
@@ -127,7 +127,7 @@ else {
 </script>
 
 <template>
-  <div class="md:pl-[5vw] font-sans text-white min-h-screen min-w-screen flex flex-col items-start justify-start">
+  <div class="md:pl-[5vw] font-sans text-white min-h-dvh min-w-screen flex flex-col items-start justify-start">
```

Comment:
this looks great but do we need to add anything for browsers that don't support dvh, e.g. a fallback? Maybe we could add a custom rule in uno config to add multiple properties including the 100vh fallback?

Reply:
Yip. We should def do that. Though it is quite widely supported now.